custom_components/waste_collection_schedule/waste_collection_schedule/wizard/irnt_no.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+import urllib.parse
+
+import inquirer
+import requests
+
+URL_ADDRESS_SEARCH = "https://innherredrenovasjon.no/wp-json/ir/v1/addresses/{search}"
```

Comment:
I really appreciate that you created a wizard to get the magic premise_id. But from my experience a lot of users have problems running such a wizards. Nowadays I would prefer to specify the address as source argument and let the source query the premise_id internally.